src/main.rs

```diff
@@ -10,3 +12,27 @@ fn main() -> Result<()> {
 
     Ok(())
 }
+
+pub struct PreBuild {}
+impl PreBuild {
+    /// When kaps will have its own release on GitHub, the binary could be fetched here.
+    /// Meanwhile, we are using git2 library to clone Kaps from GitHub, then cargo to build it from the sources.
+    pub fn build_kaps(kaps_repository_url: &str) {
+        println!("Removing old kaps folder");
```

Comment:
You should use a proper logger, but that can be done through an upcoming PR.

Reply:
Yep, this should be done in another PR.